extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -75,27 +80,46 @@ const AddResourcesSectionName = styled(Text)`
   font-size: ${FluentTheme.fonts.mediumPlus.fontSize};
 `;
 
-const labelTooltipStyles = {
+const ConfigureResourcesSectionName = styled(Text)`
+  font-size: ${FluentTheme.fonts.mediumPlus.fontSize};
```

Comment:
does it need px?

Reply:
Based on the figma.

Reply:
no, I mean this ${FluentTheme.fonts.mediumPlus.fontSize} will evaluate to '14' as a string with no px (I think)

Reply:
nvm, it seems like they are using px values as string,